src/generic_ui/polymer/invite-user.html

```diff
@@ -221,27 +215,15 @@
           </div>
 
           <div id='allNetworkButtons'>
-
-            <div id='cloudInstall' hidden?='{{ model.cloudProviderNames.length < 1 }}' on-tap='{{ cloudInstall }}'>
-              <div class='content'>
-                <img src='../../icons/cloud/connect-cloud.png'>
-                <h1>{{ 'CLOUD_INSTALL_INVITE_SCREEN_TITLE' | $$ }}</h1>
-                <p>{{ 'CLOUD_INSTALL_INVITE_SCREEN_MESSAGE' | $$ }}</p>
-              </div>
-              <div class='chevronWrapper'>
-                <core-icon icon="chevron-right"></core-icon>
-              </div>
-            </div> <!-- end of cloud -->
-
-            <div class='categoryHeader'>{{ "CHOOSE_A_SOCIAL_NETWORK" | $$ }}</div>
-
             <template repeat='{{ n in model.networkNames }}' vertical layout>
```

Comment:
bec4733 makes sure Cloud and Quiver are first and second in the list, respectively
![screen shot 2016-06-23 at 17 29 28](https://cloud.githubusercontent.com/assets/64992/16320879/1e66c02c-3968-11e6-9a30-5deab8b916f0.png)